### ISSUE TITLE
DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working

### ISSUE THREAD

**Issue by garyvdm** [sass/libsass-python#264]
When using libsass with python 3.7, one gets this warning message:

```
lib/python3.7/site-packages/sass.py:582: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  elif isinstance(custom_functions, (collections.Set, collections.Sequence)):
```

**Comment by asottile** [comment]
Thanks for the report!  working on a fix for this today!

**Comment by asottile** [comment]
This has been released as part of 0.15.0

**Comment by garyvdm** [comment]
Awesome. Thanks.
